### ISSUE TITLE
Posibility to parse positional argument?

### ISSUE THREAD

**Issue by vijayganeshpk** [unosquare/swan#223]
**Is your feature request related to a problem? Please describe.**
Not all arguments need to have short or long name prefixes, SWAN should have an option to mention the default argument that will be passed as the first one without any prefixes.

**Describe the solution you'd like**
Provide an option to mention a property of the option class as the default first position argument

```csharp
[ArgumentOption(DefaultArg=true, HelpText = "Set user name.")]
public string Username { get; set; }

[ArgumentOption("ou", HelpText = "Org. Unit")]
public string OrgUnit{ get; set; }
```
should enable usage as below:

```shell
$ myprog myusername --ou myOU
myusername would be assigned to Username property
myOU would be assigned to OrgUnit property
```

**Describe alternatives you've considered**

**Additional context**

**Comment by geoperez** [comment]
I like the idea. I'll give it a try this week.

**Comment by geoperez** [comment]
Let's keep open this issue. @ManuSanRam can you take a look?

**Comment by geoperez** [comment]
I need to release a new version including this feature.

**Comment by geoperez** [comment]
New version published, v3.1.0.
